### PR TITLE
[compat2021] Add disclaimer for STP version

### PR DESCRIPTION
**Pull request by stephenmcgruer**
See https://github.com/web-platform-tests/wpt.fyi/issues/2663

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
image

https://smcgruer-compat-2021-s-dot-wptdashboard-staging.uk.r.appspot.com/compat2021

**Comment by KyleJu** [comment]
`web_components_test` is failing due to #2677. I will dig into it shortly

**Comment by foolip** [comment]
@gsnedders do you want to review the wording here? FWIW, I don't think it's accurate to call it a data issue with wpt.fyi, but I also don't know how much detail to put in there. "Due to macOS disallowing installation of system-wide certificates with SIP enabled" would be a bit much...

**Comment by stephenmcgruer** [comment]
> FWIW, I don't think it's accurate to call it a data issue with wpt.fyi, but I also don't know how much detail to put in there.

"infrastructure issue" ?

**Comment by foolip** [comment]
Just "infrastructure issue" sounds good to me!

File: webapp/components/compat-2021.js [hunk 1]
```diff
@@ -214,6 +214,14 @@ class Compat2021 extends PolymerElement {
       </style>
       <h1>Compat 2021 Dashboard</h1>
       <compat-2021-summary stable="[[stable]]"></compat-2021-summary>
+      <template is="dom-if" if="[[showSTPWarning]]">
+        <info-banner>
+          <em>NOTE:</em> Due to a
+          <a href="https://github.com/web-platform-tests/wpt/issues/31147">data
+          issue</a> with wpt.fyi, the version of Safari Tech Preview used here
+          is significantly out of date.
```
Comment (foolip) [inline webapp/components/compat-2021.js:222]:
```suggestion
          <em>NOTE:</em> Due to an
          <a href="https://github.com/web-platform-tests/wpt/issues/31147"
          >infrastructure issue</a>, the version of Safari Technology Preview
          used here is significantly out of date.
```

**Review by foolip — DISMISSED** [review]

**Comment by stephenmcgruer** [comment]
image

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Let's ignore the web_components_test failure here.

**Pull request merged**